test/box/gh-6405-discarding-arguments-in-box-func-call.test.lua

```diff
@@ -0,0 +1,29 @@
+args_type = function(...)                     \
+    local type_table = {}                     \
+    for i = 1, select('#', ...) do            \
+        type_table[i] = type(select(i, ...))  \
+    end                                       \
+    return type_table                         \
+end                                           \
+
+box.schema.func.create('args_type')
+
+box_call = box.func.args_type:call({1, nil, 3})
+lua_call = args_type(1, nil, 3)
+assert (table.equals(box_call, lua_call))
```

Comment:
Nit: we do not put extra space before bracer.

Reply:
Fixed

Reply:
>`assert (`

No, it is not.